deploy/services/README.md

```diff
@@ -73,6 +84,17 @@ Please replace the same with your server's hostname.
   cp config/services.env.template config/services.env
   ```
 
+* Start PostgreSQL and run ThingsBoard install.
+
+  ```bash
+  docker compose -f compose.services.secure.yml\
+    --env-file config/services.env\
+    up -d postgres
+  docker compose -f compose.services.secure.yml \
+    --env-file config/services.env \
+    run --rm -e INSTALL_TB=true thingsboard-ce
+  ```
```

Comment:
thingsboard has installation step followed by service start step. Please see [docs](https://thingsboard.io/docs/user-guide/install/docker/). The service start step must be specified as well.